modules/rgbd/include/opencv2/rgbd/volume.hpp

```diff
@@ -0,0 +1,58 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+// This code is also subject to the license terms in the LICENSE_KinectFusion.md file found in this
+// module's directory
+
+#ifndef __OPENCV_VOLUME_H__
+#define __OPENCV_VOLUME_H__
+
+#include "intrinsics.hpp"
+#include "opencv2/core/affine.hpp"
+
+namespace cv
+{
+namespace kinfu
+{
+
+class Volume
+{
+   public:
+    explicit Volume(float _voxelSize, cv::Affine3f _pose, float _raycastStepFactor)
```

Comment:
> explicit 

Makes sense for cases with single parameter only.

Reply:
Done.